### PR TITLE
Improve styling of selected exemplar details

### PR DESCRIPTION
**Pull request by juliusv**
This isn't perfect yet, but IMO better than the initial draft.

https://user-images.githubusercontent.com/538008/122118900-e3281280-ce28-11eb-8d31-5f642236c10d.mp4

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@LeviHarrison Would be cool to squeeze this in last-minute before the release :smile:

**Comment by LeviHarrison** [comment]
Looking now 👀

**Comment by LeviHarrison** [comment]
Also would be great to get #8909 in before release

**Comment by juliusv** [comment]
> Also would be great to get #8909 in before release

Forgot that wasn't merged yet, thanks for the reminder! Done.

**Review by LeviHarrison — APPROVED** [review]
Looks pretty good! I think we might want to add tests for this in the future, but for now LGTM!

**Review by roidelapluie — APPROVED** [review]

File: web/ui/react-app/src/pages/graph/Graph.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { Legend } from './Legend';
 import { Metric, ExemplarData, QueryParams } from '../../types/types';
 import { isPresent } from '../../utils';
 import { normalizeData, getOptions, toHoverColor } from './GraphHelpers';
+import { Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
Comment (roidelapluie) [inline web/ui/react-app/src/pages/graph/Graph.tsx:10]: Does this load it locally or via CDN's?

Comment (juliusv) [reply web/ui/react-app/src/pages/graph/Graph.tsx:10]: Locally, we already use this icon in other places as well.

**Comment by juliusv** [comment]
Thanks :)

**Pull request merged**